Gruntfile.js

```diff
@@ -183,11 +186,13 @@ module.exports = function(grunt) {
         }
 
         eg = file.read(filepath);
-        md = "docs/" + example.file.replace(".js", ".md");
+        name = (example.name || example.file).replace(".js", "");
```

Comment:
Want to change my syntax for initializing images and breadboards to match? (currently lines 224 and 228)

``` js
images = examples.images || [];
```